src/modules/Builder/components/ToggleItemContainer/ToggleItemContainer.styles.ts

```diff
@@ -23,7 +23,10 @@ export const StyledItemOption = styled(StyledItemOptionContainer, shouldForwardP
 `;
 
 export const StylesTitleWrapper = styled(StyledFlexTopCenter, shouldForwardProp)`
-  ${({ open }: { open?: boolean }) => `
-		height: ${open ? '5.6rem' : 'inherited'}; 
+  justify-content: space-between;
+
+  ${({ open, uiType }: { open?: boolean; uiType: ToggleContainerUiType }) => `
+		height: ${open ? '5.6rem' : 'inherit'}; 
```

Comment:
smth with spaces?

Reply:
In IDE everything looks good:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/dcddaf25-e81e-4160-8922-89005486942c)